### PR TITLE
Add support for `brand`, `flexible_disk` and `disks` for packages (#182)

### PR DESCRIPTION
**Pull request by kusor**

### CODE REVIEW

File: compute/packages_test.go [hunk 1]
```diff
@@ -50,7 +50,29 @@ func TestAccPackagesList(t *testing.T) {
...
 					testPackageID = pkgs[0].ID
 					testPackageName = pkgs[0].Name
-
+					for _, pkg := range pkgs {
+						if len(pkg.Disks) != 0 {
+							if pkg.Brand != "bhyve" {
+								t.Fatalf("Package with disks does not have brand \"bhyve\": %+v", pkg.Disks)
+							}
+							for i := 0; i < len(pkg.Disks); i++ {
+								dsk := (*pkg).Disks[i]
```
Comment (twhiteman) [inline compute/packages_test.go:59]: Does this need the `(*pkg)` here?

Comment (kusor) [reply compute/packages_test.go]: No, it doesn't. Removed!

**Comment by kusor** [comment]
> You could also look at extending the Mock package (see listPackagesSuccess) to add disks, or add another simple unit test that tests a package with disks.

Good idea. Added now.

File: compute/packages_test.go [hunk 2]
```diff
@@ -50,7 +50,29 @@ func TestAccPackagesList(t *testing.T) {
...
 					testPackageID = pkgs[0].ID
 					testPackageName = pkgs[0].Name
-
+					for _, pkg := range pkgs {
+						if len(pkg.Disks) != 0 {
+							if pkg.Brand != "bhyve" {
+								t.Fatalf("Package with disks does not have brand \"bhyve\": %+v", pkg.Disks)
+							}
+							for i := 0; i < len(pkg.Disks); i++ {
+								dsk := pkg.Disks[i]
```
Comment (twhiteman) [inline compute/packages_test.go:59]:
```suggestion
							for _, dsk := range pkg.Disks {
```

**Pull request merged**